geomstats/stiefel.py

```diff
@@ -165,7 +161,7 @@ def exp(self, tangent_vec, base_point):
 
         return exp
 
-    def log(self, point, base_point, max_iter=100, tol=1e-6):
+    def log(self, point, base_point, max_iter=30):
```

Comment:
did you change this because it was taking too long?

Reply:
Yes, because I deleted the early stopping condition because of the lack of easy if statement in tf.